ui/enhancement_shaman/_sim.scss

```diff
@@ -0,0 +1,23 @@
+// Set color variables which control the site theme.
+.elemental-shaman-sim-ui {
```

Comment:
This should be 'enhancement-shaman-sim-ui' to match the value in sim.ts. This is probably why your colors aren't working.